app/uk/gov/hmrc/tai/config/ApplicationConfig.scala

```diff
@@ -87,6 +87,7 @@ class MongoConfig @Inject()(val runModeConfiguration: Configuration) extends Bas
   lazy val mongoEnabled: Boolean = runModeConfiguration.getOptional[Boolean]("cache.isEnabled").getOrElse(false)
   lazy val mongoEncryptionEnabled: Boolean =
     runModeConfiguration.getOptional[Boolean]("mongo.encryption.enabled").getOrElse(true)
+  lazy val mongoTTL: Int = runModeConfiguration.getOptional[Int]("cache.expiryInMinutes").getOrElse(0)
```

Comment:
Default should be 15

Reply:
Done, though thinking about it wouldn't the time be in seconds? I think it was before by default